### ISSUE TITLE
Latest colorlog release breaks macsydata on fresh macsyfinder installs

### ISSUE THREAD

**Issue by Alexandre-Herve** [gem-pasteur/macsyfinder#38]
## Version of MacsyFinder:

Macsyfinder 2.0rc4
using:
- Python 3.8.10 (default, Jun  2 2021, 10:49:15)  [GCC 9.4.0]
- NetworkX 2.6.2

### OS

- [x] Linux
- [ ] Windows
- [ ] Mac

## Expected behavior

Macsydata works normally after fresh install

## Actual behavior

The [latest release of colorlog](https://pypi.org/project/colorlog/6.4.1/) brought a breaking change to their API, which breaks new installs from MacSyFinder: macsydata doesn't work anymore.

## Steps to reproduce behavior

```
conda create -n macsyfinder python pip=20.3.3
conda activate macsyfinder
pip install macsyfinder
macsydata list
```

## Relevant logs and/or screenshots
```
Traceback (most recent call last):
  File "/home/alexandre/miniconda3/envs/macsyfinder/bin/macsydata", line 8, in <module>
    sys.exit(main())
  File "/home/alexandre/miniconda3/envs/macsyfinder/lib/python3.9/site-packages/macsypy/scripts/macsydata.py", line 824, in main
    macsypy.init_logger()
  File "/home/alexandre/miniconda3/envs/macsyfinder/lib/python3.9/site-packages/macsypy/__init__.py", line 48, in init_logger
    logging = colorlog.logging.logging
AttributeError: module 'colorlog' has no attribute 'logging'

```

**Comment by bneron** [comment]
fixed with commit 7b73a529

**Comment by Alexandre-Herve** [comment]
Awesome thanks!
